web/styles/main.css

```diff
@@ -1 +1,15 @@
+
+.td{
```

Comment:
Main.css is an auto-generated file when using less. http://lesscss.org/usage/ Did you manually add the .td and .form-control-sm? Please add it in I would assume the _profile.less file. Then when you compile using lessc main.less main.css all of the CSS in the less files will be included.

Reply:
I think the best way to stop us from having issues with Main.css from now on is to add it to the .gitignore. Can you please do that in your next commit. We will mention in the README.md for the user to run npm less to fix the styles. In the web/package.json we would need "less": "lessc styles/main.less styles/main.css" added to the scripts.

Reply:
I tried using less to no avail. Let's resolve this problem at a later time since it's a styling issue.